conda_build/metadata.py

```diff
@@ -448,6 +448,7 @@ def parse(data, config, path=None):
         'name': None,
         'version': None,
         'number': None,
+        'entry_points': None,
```

Comment:
But actually shouldn't this be:
```suggestion
        'entry_points': list,
```

Reply:
It seems all the others here are `None`. IIUC this is because the `outputs` fields are not required since some recipes won't use `outputs`. If we make this `list`, wouldn't that imply `outputs/entry_points` is required by all recipes (even when other `outputs` fields are not)? Admittedly I could be misunderstanding things

Reply:
I don't think specifying it as a `list` makes it required but you may be right for other reasons, [I remember now how the schema validator we use here is incomplete and struggles with nested stuff](https://github.com/conda/conda-build/pull/4179). So I'll retract my comment and we'll leave it as `None`.